.github/workflows/mkl.yaml

```diff
@@ -0,0 +1,116 @@
+name: MKL Algebra
+
+on:
+  push:
+    branches: [ master, develop**, ci ]
+    tags:
+      - '*'
+  pull_request:
+    branches: [ master, develop** ]
+
+jobs:
+
+  build_and_test:
+      runs-on: ${{ matrix.os }}
+
+      strategy:
+        fail-fast: false
+
+        matrix:
+          # Specify the exact Windows image because the CMake generator in the CI breaks when the image
+          # updates (since Visual Studio is also updated at the same time).
+          os: [ubuntu-latest, macos-latest, windows-2022]
+          python-version: [3.9]
+          long: ['ON', 'OFF']
+
+          # MKL only supports double precision in the iterative solver, so that is the only combination we support as well.
```

Comment:
Perhaps we add `float: ['OFF']` as a single matrix entry to go right next to this comment and use it down below? Just so it remains close to the part of the CI if affects (currently).

Reply:
Good idea, that way it better mimics the other CI workflows. Just pushed that change.